### PR TITLE
Add API diff between .NET Core 2.2 and .NET Core 3.0 preview1.

### PR DESCRIPTION
**Pull request by ahsonkhan**
**Generated from the following two commits (including System namespaces only):**
Oct 30 (.NET Core 2.2) - https://github.com/dotnet/corefx/commit/694e15f9974011662b3d5d17b2e3c248b33b527e
Nov 21 (.NET Core 3.0 preview1) - https://github.com/dotnet/corefx/commit/57358b7f7214c233e8397483a9afc1698ac00c5b

I only included what's in-box based on previous API diffs.

cc @richlander, @terrajobst, @leecow 

Please do a sanity check for your feature areas.

cc @stephentoub, @tannergooding, @karelz, @JeremyKuhne, @steveharter, @GrabYourPitchforks, @tarekgh, @maryamariyan, @bartonjs

### CODE REVIEW

File: release-notes/3.0/preview/api-diff/preview1/3.0-preview1_System.Numerics.Tensors.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+# System.Numerics.Tensors
+
+``` diff
++namespace System.Numerics.Tensors {
```
Comment (tannergooding) [inline release-notes/3.0/preview/api-diff/preview1/3.0-preview1_System.Numerics.Tensors.md]: I don't believe this is marked as shipping yet.

File: release-notes/3.0/preview/api-diff/preview1/3.0-preview1_System.Runtime.Intrinsics.X86.md [hunk 2]
```diff
@@ -0,0 +1,673 @@
+# System.Runtime.Intrinsics.X86
+
+``` diff
+ namespace System.Runtime.Intrinsics.X86 {
```
Comment (tannergooding) [inline release-notes/3.0/preview/api-diff/preview1/3.0-preview1_System.Runtime.Intrinsics.X86.md]: Not sure if it's worth mentioning, but for the HWIntrinsics, the snap happened in a somewhat irregular place and there are several APIs (basically the delta between preview1 and preview2) that no longer exist and will always throw PNSE when called (if on preview1).

**Comment by tarekgh** [comment]
@ahsonkhan are we going to do the same after preview 3 too?

**Comment by ahsonkhan** [comment]
> are we going to do the same after preview 3 too?

I see no reason why we can't or shouldn't.

File: release-notes/3.0/preview/api-diff/preview1/3.0-preview1_System.Net.Sockets.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# System.Net.Sockets
+
+``` diff
+ namespace System.Net.Sockets {
+     public class NetworkStream : Stream {
+-        public override int Read(Span<byte> destination);
++        public override int Read(Span<byte> buffer);
+-        public override void Write(ReadOnlySpan<byte> source);
```
[the remark this answers is not in the document]
Comment (ahsonkhan) [reply release-notes/3.0/preview/api-diff/preview1/3.0-preview1_System.Net.Sockets.md]: You are correct. I was comparing the implementation assemblies not the ref assemblies.

https://github.com/dotnet/corefx/blob/29e580bb69bbd0bc857e28bbc905ede6c8e41d3b/src/System.Net.Sockets/src/System/Net/Sockets/NetworkStream.cs#L308

File: release-notes/3.0/preview/api-diff/preview1/3.0-preview1_System.Diagnostics.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
+# System.Diagnostics
+
+``` diff
+ namespace System.Diagnostics {
+     public class Activity {
+-        public static Activity Current { get; private set; }
```
[the remark this answers is not in the document]
Comment (ahsonkhan) [reply release-notes/3.0/preview/api-diff/preview1/3.0-preview1_System.Diagnostics.md]: True, but that's how API reviewer shows the diff. I can either leave it as is to be consistent with previous diffs, manually change this one instance, or we update how API reviewer does diffing. Which option would you prefer?

cc @terrajobst

**Comment by ahsonkhan** [comment]
@stephentoub, can you please take another look?

**Comment by Anipik** [comment]
cc @terrajobst

**Pull request merged**